src/v2/guide/reactivity.md

```diff
@@ -4,80 +4,80 @@ type: guide
 order: 601
 ---
 
-Now it's time to take a deep dive! One of Vue's most distinct features is the unobtrusive reactivity system. Models are just plain JavaScript objects. When you modify them, the view updates. It makes state management simple and intuitive, but it's also important to understand how it works to avoid some common gotchas. In this section, we are going to dig into some of the lower-level details of Vue's reactivity system.
+Bây giờ chúng ta sẽ cùng tìm hiểu sâu hơn về Vue! Một trong những tính năng khác biệt nhất của Vue là hệ thống phản ứng "không phô trương". Các models chỉ là các đối tượng Javascript thuần. Khi bạn chỉnh sửa chúng thì view sẽ thay đổi. Nó làm cho việc quản lý trạng thái đối tượng được đơn giản và trực quan hơn, nhưng điều quan trọng là vẫn nên hiểu cách hoạt động của nó để tránh một số sai lầm phổ biến. Trong phần này, chúng ta sẽ đi sâu vào một số chi tiết ở mức thấp hơn của hệ thống phản ứng của Vue.
 
-## How Changes Are Tracked
+## Các thay đổi trong Vue được theo dõi như thế nào
 
-When you pass a plain JavaScript object to a Vue instance as its `data` option, Vue will walk through all of its properties and convert them to getter/setters using [Object.defineProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty). This is an ES5-only and un-shimmable feature, which is why Vue doesn't support IE8 and below.
+Khi bạn truyền một đối tượng thuần JavaScript cho một thực thể Vue như đối tượng `data` của nó, Vue sẽ duyệt qua tất cả các thuộc tính của nó và chuyển đổi chúng thành các getter / setters sử dụng [Object.defineProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty). Đây là một tính năng chỉ có ở ES5 và khó để thay thế, đó là lý do tại sao Vue không hỗ trợ IE8 và các phiên bản thấp hơn của nó.
 
-The getter/setters are invisible to the user, but under the hood they enable Vue to perform dependency-tracking and change-notification when properties are accessed or modified. One caveat is that browser consoles format getter/setters differently when converted data objects are logged, so you may want to install [vue-devtools](https://github.com/vuejs/vue-devtools) for a more inspection-friendly interface.
+Các getter và setter có thể được nhìn thấy bởi người dùng, nhưng sâu hơn nữa, nó cho phép Vue thực hiện theo dõi các phụ thuộc và thay đổi thông báo khi các thuộc tính được truy cập hoặc sửa đổi. Một trong những hạn chế của việc này là trình duyệt set in ra định dạng getter / setters khác nhau khi chuyển đổi các đối tượng, do đó khuyến khích các bạn cài đặt [vue-devtools](https://github.com/vuejs/vue-devtools) để có một cái nhìn sâu hơn về ứng dụng của mình.
```

Comment:
* "invisible to the user" nghĩa là "Người dùng _không thấy được_" chứ không phải "có thể được nhìn thấy.
* "under the hood" ở đây nghĩa là bên trong, nôm na là "đằng sau hậu trường"
* "[..] change-notification" không phải là "thay đổi thông báo" mà là "thông báo (các) thay đổi"
* trình duyệt _sẽ_ in ra? Nhưng đây không phải là trình duyệt in ra, mà là console của trình duyệt.
* "do đó _chúng tôi_ khuyến khích các bạn cài đặt" hoặc "các bạn nên cài đặt," nếu không thì câu này sẽ không có chủ ngữ